ad-joining/register-computer/main.py

```diff
@@ -289,6 +323,41 @@ def __register_computer(request):
         logging.exception("Checking project access to '%s' failed" % auth_info.get_project_id())
         return flask.abort(HTTP_ACCESS_DENIED)
     
+    # If custom OU is being used, make sure all provided OUs (root and computer) are valid    
+    if "CUSTOM_OU_ROOT_DN" in os.environ:
```

Comment:
We might want to split up this function a bit -- it has grown very long

Reply:
Fixed.